src/replica/storage/simple_kv/tests/case.cpp

```diff
@@ -39,9 +39,15 @@
 
 #include <dsn/tool-api/task.h>
 #include <dsn/tool-api/rpc_message.h>
+<<<<<<< HEAD:src/replica/storage/simple_kv/tests/case.cpp
```

Comment:
Conflicts need to be resolved.

Reply:
I am sorry I didn't found it yesterday. fixed